workflow/djworker.py

```diff
@@ -0,0 +1,150 @@
+import datajoint as dj
+import inspect
+import time
+import os
+import platform
+from datetime import datetime
+
+
+_populate_settings = {
+    'display_progress': True,
+    'reserve_jobs': True,
+    'suppress_errors': True}
+
+
+class WorkerLog(dj.Table):
+    definition = """
+    # Registration of processing jobs running .populate() jobs (e.g. in process.py script)
+    process_timestamp : datetime      # timestamp of the processing job
+    process           : varchar(64)
+    ---
+    worker_name=''    : varchar(255)  # name of the worker
+    host              : varchar(255)  # system hostname
+    user=''           : varchar(255)  # database user
+    pid=0             : int unsigned  # system process id  
+    """
+
+    _table_name = '~WorkerLog'
+
+    @classmethod
+    def log_process_job(cls, process, worker_name='', db_prefix=''):
+        if isinstance(process, dj.user_tables.TableMeta):
+            schema_name, table_name = process.full_table_name.split('.')
+            schema_name = schema_name.strip('`').replace(db_prefix, '')
+            table_name = dj.utils.to_camel_case(table_name.strip('`'))
+            process_name = f'{schema_name}.{table_name}'
+            user = process.connection.get_user()
+        elif inspect.isfunction(process):
+            process_name = process.__name__
+            user = ''
+
+        if not worker_name:
+            frame = inspect.currentframe()
+            function_name = frame.f_back.f_code.co_name
+            module_name = inspect.getmodule(frame.f_back).__name__
+            worker_name = f'{module_name}.{function_name}'
+
+        cls.insert1({'process': process_name,
+                     'process_timestamp': datetime.utcnow(),
+                     'worker_name': worker_name,
+                     'host': platform.node(),
+                     'user': user,
+                     'pid': os.getpid()})
+
+    @classmethod
+    def print_recent_jobs(cls, backtrack_minutes=60):
```

Comment:
This function does not print.  So change the name.
```suggestion
    def recent_jobs(cls, backtrack_minutes=60):
```